### DISCUSSION TITLE
Images issue after update

### DISCUSSION THREAD

**Discussion by Soren90** [gohugo-ananke/ananke#807] [2 - Q&A / Support] [answered]
Hi!

I'm trying to update my ananke theme from v2.6.1 -> v2.11.2 and I've seen some great improvements to the theme.

Current setup: https://github.com/Soren90/legacyligan/tree/main
Updated setup: https://github.com/Soren90/legacyligan/tree/ananke-v2.11.2-update

After I've done the update, it seems like the archive preview on the index site adds the pictures from post/*.md rather than just showing the featured_image as it did before. The same goes for $BASEURL/post, it adds the pictures and kind of borks the site. I've not been able to narrow down where this behavior has changed. Could someone please give me a pointer?

The current site look like this:
![image](https://github.com/user-attachments/assets/06685f20-0702-490e-a9e7-a36ecf4b277e)

And after update it looks like this:
![image](https://github.com/user-attachments/assets/057b42f1-688b-4e41-b1b0-2a82249d26da)

**Comment by davidsneighbour** [comment, marked as answer]
I am not sure if I understand your issue correct... 

Hugo itself started in https://github.com/gohugoio/hugo/releases/tag/v0.134.0 to render the preview from the rendered HTML (x words, then extend to the paragraphs boundaries, then cut this and use as summary). This is not something I will not be able to fix within the layout files themselves I am afraid.

You could downgrade Hugo to https://github.com/gohugoio/hugo/releases/tag/v0.133.1 to return to the previous summary behaviours. but this will lead to issues down the road with updates to the theme and new Hugo features.

You need to edit what your list pages show as summary by adding a `summary` frontmatter to EACH post. That is much work, but currently the only way to get back to the previous behaviour. The theme will show what is in `summary` and fall back to the default GoHugo summary behaviour. 

You don't need to add a summary to all posts, just the ones where GoHugo goes berserk. Or you could move the actual content of your posts further down and write a less extended intro paragraph, but that is editorial hell... I am not happy with this change in GoHugo, but it does streamline everything, because from three ways to get the summary two always were rendered and one not which seems to have been leading to issues.

**Reply by Soren90** [reply]
Thank you for taking your time to reply. I'm still pretty new to Hugo and I've found it pretty hard to troubleshoot when you go into the nitty gritty of it. But I'll take a closer look on how to add summarys. Thanks for that.

Now that I've been thinking more about it, what I want is basically to ignore pictures on _index.md and in /post. The only place i want images to show up is when you click the read more button or actually visit the article in itself. Do you know if that's possible?

**Reply by Soren90** [reply]
I did find a workaround though: https://gohugo.io/content-management/summaries/

You can use:
```
<!--more-->
```
Then everything under will basically not be a part of the summary. This at least solves the issue. But it would be optimal if summaries basically just ignores pictures.

**Reply by davidsneighbour** [reply]
The `<!--more-->` solution is one of those three summary solutions that I always forget. Yep, GoHugo will take everything before the `<!--more-->` for the overview page.

Regarding the general removal of images from the summaries - that is something that might be possible, but only with your own list-layout. You could take the summary and then via regexp take the image tags out. But that is very hacky, so the `<!--more-->` solution is probably much better.
